### PR TITLE
Refine the unit test of convolution function.

### PR DESCRIPTION
**Pull request by hedaoyuan**

### CODE REVIEW

**Comment by NHZlX** [comment]
More and more test files are added to paddle/function, for users, it looks very messy, should we consider to building a folder ?

**Comment by NHZlX** [comment]
The refinement of the test of convolution function looks great to me.

**Comment by hedaoyuan** [comment]
> More and more test files are added to paddle/function, for users, it looks very messy, should we consider to building a folder ?

Put the test file and source file in one directory is a new convention. You can see in the `platform`, `operators`, `framework` the test file and source file are also placed in a directory.

File: paddle/function/nnpack/NNPACKConvOp.cpp [hunk 1]
```diff
@@ -196,10 +196,13 @@ class NNPACKConvFunction : public ConvFunctionBase {
         CHECK_EQ(status, nnp_status_success);
       }
     } else {
+      // only supports stride = 1
+      CHECK_EQ(strideH(), 1);
+      CHECK_EQ(strideW(), 1);
+
+      // TODO(hedaoyuan): There has some bug when batchSize > 1 and groups_ > 1.
+      CHECK_EQ(groups_, (size_t)1);
```
Comment (wangkuiyi) [inline paddle/function/nnpack/NNPACKConvOp.cpp]: `(size_t)1` ==> `size_t(1)` or `static_cast<size_t>(1)`, according to Google C++ style https://google.github.io/styleguide/cppguide.html#Casting

Comment (hedaoyuan) [inline paddle/function/nnpack/NNPACKConvOp.cpp]: Thanks for pointing out. Done.

**Pull request merged**